model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -219,29 +219,6 @@
     LLMInferenceFramework.TENSORRT_LLM: [],
 }
 
-# We need a dict where if we need to override we can
```

Comment:
what's the reason we're getting rid of the `max_model_len`/`max_num_batched_tokens` args to vllm?

Reply:
i forgot why we're doing this in the first place, but i'm pretty certain in recent version of vLLM this is not needed. also checked most of these models, config.json has the same `max_position_embeddings` with the values here